pages/index.tsx

```diff
@@ -27,18 +38,81 @@ const Blurb = styled.div`
 
 export default function Index() {
   const [tabIndex, setTabIndex] = useState(0);
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const options = [
+    "General Info",
+    "Manage Position",
+    "All Positions",
+    "Wrap/Unwrap WETH",
+    "Yield Calculator",
+  ];
+
+  const handleClickListItem = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleMenuItemClick = (
+    event: React.MouseEvent<HTMLElement>,
+    index: number
+  ) => {
+    setSelectedIndex(index);
+    setAnchorEl(null);
+    setTabIndex(index);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
   return (
     <Container maxWidth={"md"}>
       <Box py={4}>
         <Header />
         <EmpSelector />
-        <StyledTabs value={tabIndex} onChange={(_, i) => setTabIndex(i)}>
-          <Tab label="General Info" disableRipple />
-          <Tab label="Manage Position" disableRipple />
-          <Tab label="All Positions" disableRipple />
-          <Tab label="Wrap/Unwrap WETH" disableRipple />
-          <Tab label="Yield Calculator" disableRipple />
-        </StyledTabs>
+        <Hidden only={["sm", "xs"]}>
+          <StyledTabs value={tabIndex} onChange={(_, i) => setTabIndex(i)}>
+            {options.map((option, index) => (
+              <Tab label={option} disableRipple />
+            ))}
+          </StyledTabs>
+        </Hidden>
+        <Hidden only={["md", "lg", "xl"]}>
+          <div>
+            <Box pt={1} pb={2}>
+              <Grid container spacing={2}>
+                <Grid item>
+                  <Button variant="contained" onClick={handleClickListItem}>
+                    Menu
```

Comment:
Thoughts on changing this to an IconButton using the "menu" icon from https://material-ui.com/components/material-icons/ ?

Reply:
ye, I'll add a burger :)

Reply:
done 
![image](https://user-images.githubusercontent.com/12886084/88548891-f3bc7800-d01f-11ea-821f-ba464f3ff963.png)

Reply:
You like this one over Menu more?